app/models/item.rb

```diff
@@ -1,5 +1,6 @@
 class Item < ActiveRecord::Base
-  # has_attached_file :photo, :default_url => "/images/:style/default.jpg"
+  has_attached_file :image, :default_url => "default.jpg"
+  validates_attachment_content_type :image, content_type: ['image/jpg', 'image/jpeg', 'image/png']
```

Comment:
Line is too long. [98/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.